index.html

```diff
@@ -26,6 +26,14 @@ <h1 id="logo">Aziz Mejri</h1>
         </ul>
       </nav>
     </header>
+    <nav id="mobile-menu" class="mobile-menu">
+      <button id="x-button" type="button">✕</button>
+      <ul>
+        <li><a class="nav-link-mobile" href="#projects">Portoflio</a></li>
+        <li><a class="nav-link-mobile" href="#about-me">About</a></li>
+        <li><a class="nav-link-mobile" href="#contackt">Contact</a></li>
```

Comment:
- [ ] Great job implementing the changes requested by the previous reviewer :+1: 
However, I have noticed that you failed to implement one of the project requirements for the contacts nav link. I have attached a screenshot below. This is because of the spelling mistake you made In the link of the `href` attribute. Please do well to fix it so all the project requirements are met and your page can function correctly :100: 
![image](https://user-images.githubusercontent.com/102699958/211041207-b9488838-ad38-4576-ae55-ec6eb0c3b51d.png)